apps/build-onchain-apps/src/hooks/useOnchainCoffeeMemos.ts

```diff
@@ -12,19 +11,22 @@ import type { CoffeeMemo } from '../types';
  * @returns The memos and a function to refetch them.
  */
 function useOnchainCoffeeMemos() {
+  // TODO: this state should be cleared on network change, otherwise memos from multiple chains
+  // could end up in the same list
   const [memos, setMemos] = useState<CoffeeMemo[]>([]);
 
   const handleSuccess = useCallback((newMemos: CoffeeMemo[]) => {
     setMemos(newMemos);
   }, []);
 
+  const contract = useBuyMeACoffee();
```

Comment:
Should we standardize the prefix for those Hooks, like `useContractBuyMeCoffee`?